app/routes/provider.js

```diff
@@ -0,0 +1,44 @@
+import Ember from 'ember';
+import config from 'ember-get-config';
+
+const providers = config.Reviews.providers.slice(1);
+const providerIds = providers.map(p => p.id);
+
+/**
+ * @module ember-osf-reviews
+ * @submodule routes
+ */
+
+/**
+ * @class Provider Route Handler
+ */
+export default Ember.Route.extend({
+    theme: Ember.inject.service(),
+
+    beforeModel(transition) {
+        const {slug = ''} = transition.params.provider;
+        const slugLower = slug.toLowerCase();
+
+        if (providerIds.includes(slugLower)) {
```

Comment:
I would rather not have a hard-coded list of providers, so we don't have to update code whenever new providers are added. This looks like it was copied from preprints to support resolving GUIDs... do we need a `beforeModel` at all? Seems like this route should just have a `model` method that gets the provider by id.